core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1886,6 +1886,38 @@ public String toString() {
           .setDescription("The number of workers to run on an Alluxio host for YARN framework.")
           .build();
 
+  //
+  // JVM Monitor related properties
+  //
+  public static final PropertyKey JVM_MONITOR_WARN_THRESHOLD_MS =
+      new Builder(Name.JVM_MONITOR_WARN_THRESHOLD_MS)
+          .setAlias(new String[]{"alluxio.jvm.monitor.warn.threshold.ms"})
+          .setDefaultValue("10000ms")
+          .setDescription("Extra sleep time longer than this threshold, log WARN.")
+          .build();
+  public static final PropertyKey JVM_MONITOR_INFO_THRESHOLD_MS =
+      new Builder(Name.JVM_MONITOR_INFO_THRESHOLD_MS)
+          .setAlias(new String[]{"alluxio.jvm.monitor.info.threshold.ms"})
+          .setDefaultValue("1000ms")
+          .setDescription("Extra sleep time longer than this threshold, log INFO.")
+          .build();
+  public static final PropertyKey JVM_MONITOR_SLEEP_INTERVAL_MS =
+      new Builder(Name.JVM_MONITOR_SLEEP_INTERVAL_MS)
+          .setAlias(new String[]{"alluxio.jvm.monitor.sleep.interval.ms"})
```

Comment:
These aliases are not required, we use them for backward compatibility, but since this is a new configuration option, we don't need to support the `.ms` versions.